include/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 add_subdirectory(clr)
 add_subdirectory(Driver)
+add_subdirectory(CoreDisTools)
```

Comment:
I know you're the one that adds this. But all of this additions including this CmakeLists.txt are useless. If you could clean up these since you're here, it would be great. Or at least let's not add this anymore including the empty CmakeLists.txt under CoreDisTools.